webapp/views/wpt-results.js

```diff
@@ -226,8 +226,13 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
                     </template>
                   </template>
                 </td>
-                <td>
```

Comment:
Does this feature work on subtests? if so we have to add the same code to [`test-files-results-table`](https://github.com/web-platform-tests/wpt.fyi/blob/main/webapp/components/test-file-results-table.js#L140)

Reply:
It does not.